carnavi_lidar_node:
    ros__parameters:
        lidar_ip: "192.168.123.31"
        multicast_group_ip: "224.0.0.5"
        port: 5000
        topic:
            ch0: "/scan/ch00"
            ch1: "/scan/ch01"
            frame: "base_laser_link"
        scan: 
            pub_queue_size: 15
            hz: 15
            increment: 0.25
            angle_max: 60.0
            angle_min: -60.0
            range_max: 70.0
            range_min: 0.25
